sdk/translation/azure-ai-translation-text/tests/test_break_sentence.py

```diff
@@ -0,0 +1,77 @@
+# ------------------------------------
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT License.
+# ------------------------------------
+
+from devtools_testutils import recorded_by_proxy
+from azure.ai.translation.text.models import InputTextItem
+from preparer import TextTranslationPreparer
+from testcase import TextTranslationTest
+
+class TestBreakSentence(TextTranslationTest):
+
+    @TextTranslationPreparer()
+    @recorded_by_proxy
+    def test_autodetect(self, **kwargs):
```

Comment:
It might be good to add some tests for the async client / methods as well (not blocking this PR)

Reply:
ACK, we will add them before GA